packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -2377,9 +2419,28 @@ export class Grapher
         autorun(() => (document.title = this.currentTitle))
     }
 
+    @action.bound private setUpWindowResizeEventHandler(): void {
+        const updateWindowDimensions = (): void => {
+            this.windowInnerWidth = window.innerWidth
+            this.windowInnerHeight = window.innerHeight
+        }
+        const onResize = debounce(updateWindowDimensions, 400, {
+            leading: true,
+        })
```

Comment:
For the reviewers: I'm not confident `leading=true` makes sense here?

Reply:
Okay, cool, thanks for verifying! I always get a little confused about debounce options